locales/en-US/app.ftl

```diff
@@ -993,6 +987,53 @@ TransformNavigator--collapse-function-subtree = Collapse subtree: { $item }
 #   $item (String) - Search filter of the markers that transform will apply to.
 TransformNavigator--drop-samples-outside-of-markers-matching = Drop samples outside of markers matching: “{ $item }”
 
+## "Dropped functions" settings UI
+##
+## A "dropped function" is a function which is excluded from the sample data.
```

Comment:
A comment this long is not helpful for localization, it's going to clutter the UI given its size. Comments on the individual strings are OK.